android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -1473,24 +1473,9 @@ public static boolean addKeyboard(Context context, Keyboard keyboardInfo) {
     return KeyboardPickerActivity.addKeyboard(context, keyboardInfo);
   }
 
-  // Intend to deprecate in Keyman 15.0
+  // This API is deprecated in Keyman 17.0
```

Comment:
We should just remove the function altogether rather than leaving a broken stub. That way, devs still using this API function will get a compile-time failure rather than a mysterious runtime failure.